simple_app/utils.py

```diff
@@ -111,7 +134,16 @@ class SearchForm(FlaskForm):
     """
     Searchbar class
     """
-    search = StringField('Search for an object:', id='autocomplete')  # searchbar
+    search = StringField('Search for an object:', id='mainsearchfield')  # searchbar
+    refsearch = StringField('Filter by reference or comment:', id='refsearchfield')
+    submit = SubmitField('Query', id='querybutton')  # clicker button to send request
+
+
+class LooseSearchForm(FlaskForm):
+    """
+    Searchbar class
+    """
+    search = StringField('Search for anything:', id='mainsearchfield')  # searchbar
```

Comment:
I'd be a bit more descriptive here. Maybe something like: `Search all columns in all tables for:`